src/js/pages/Profile/components/Offer.jsx

```diff
@@ -6,21 +6,26 @@ import {TokenImages} from '../../../utils/images';
 import {calculateEscrowPrice} from '../../../utils/transaction';
 
 
-const Offer = ({offer, prices, onClick}) => (
+const Offer = ({offer, prices, onClick, disabled}) => (
   <Row className="border py-2 mx-0 my-2 rounded">
     <Col xs="4" className="v-align-center">
       <p className="font-weight-bold"><img src={TokenImages[`${offer.token.symbol}.png`] || TokenImages[`generic.png`]} alt="asset icon" className="mr-2"/>{offer.token.symbol}</p>
     </Col>
     <Col xs="8" className="v-align-center text-right">
-      <Button color="primary p-2" onClick={onClick}>Buy for {truncateTwo(calculateEscrowPrice(offer, prices))} {offer.currency}</Button>
+      <Button color={disabled ? "secondary p-2" : "primary p-2" } disabled={disabled} onClick={onClick}>Buy for {truncateTwo(calculateEscrowPrice(offer, prices))} {offer.currency}</Button>
```

Comment:
You cannot put `p-2` in color lol.

Reply:
oops!  I did not read the name of the attribute!